libs/zlog/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=zlog
+PKG_VERSION:=1.2.15
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/HardySimpson/$(PKG_NAME)/archive/refs/tags/
```

Comment:
No, this will generate clashes in dl dir
See https://github.com/openwrt/packages/blob/master/multimedia/gerbera/Makefile#L14

Reply:
@diizzyy tnx, new trick up my sleeve. Updated